WORKSPACE

```diff
@@ -109,12 +109,6 @@ go_repository(
     tag = "v1.3.1",
 )
 
-go_repository(
-    name = "io_k8s_sigs_kustomize",
-    importpath = "sigs.k8s.io/kustomize",
-    tag = "v1.0.11",
-)
-
```

Comment:
Are all the bazel targets that were using this removed?

Reply:
No, I need to fix that up but I've run out of time for the day